### PR TITLE
Enhance and add support for 4.7.2 and 4.8

### PR DESCRIPTION
**Pull request by damfle**
- More recent package
- Enabled on all supported Windows' versions

Include PR #97

### CODE REVIEW

File: libraries/v4_helper.rb [hunk 1]
```diff
@@ -78,7 +79,7 @@ def feature_setup
         # Windows 10 v1703 (RS2)
         when '10.0.15063' then %w[4.0 4.5 4.5.1 4.5.2 4.6 4.6.1 4.6.2 4.7]
         # Windows 10 & Server 2016 v1709 (RS3)
-        when '10.0.16299' then %w[4.0 4.5 4.5.1 4.5.2 4.6 4.6.1 4.6.2 4.7 4.7.1]
+        when '10.0.16299' then %w[4.0 4.5 4.5.1 4.5.2 4.6 4.6.1 4.6.2 4.7 4.7.1 4.7.2]
```
Comment (Annih) [inline libraries/v4_helper.rb:82]: Are you sure? it seems to me that 4.7.2 has been introduced as Windows Feature with 1803.
What is your source? :D

Also why do you keep package setup for 1709 (L128 to L132) if you add it to feature setup?
I really think only package setup should be kept

File: libraries/package_helper.rb [hunk 2]
```diff
@@ -219,6 +225,20 @@ def packages
...
+          checksum: 'b029cfe1d1995009c856a147560b35ed858faf1f300fd42c0a0b4e0df99bfcfa',
+          not_if:   %w[KB4073120],
+        },
+        'KB4512494' => {
+          name:     'Update for Microsoft Windows (KB4512494)', # Cosmetic name
+          url:      'http://download.windowsupdate.com/d/msdownload/update/software/updt/2019/08/windows10.0-kb4512494-x64_07c10d46847e75c4267ce5cfe376265b6a87ed78.msu',
+          options:  '/norestart /quiet',
+          checksum: 'dc5865db807a5f94ab39da8257c2f694bfead3d5c9ff05b623653fb8ff874633',
+          not_if:   %w[KB4512494],
+        },
```
Comment (Annih) [inline libraries/package_helper.rb:241]: You don't seem to reference KB4512494 anywhere, why adding it?

**Review by Annih — CHANGES_REQUESTED** [review]
I would extract the addition of the 4.7.2 patch out of this PR it does not make a real coherency to me.

File: libraries/package_helper.rb (hunk 2 above)
Comment (damfle) [reply libraries/package_helper.rb:241]: was in PR, fix a bug for drm-ed content with IE, likely unecessary, will remove

**Comment by damfle** [comment]
> I would extract the addition of the 4.7.2 patch out of this PR it does not make a real coherency to me.

I kept most of PR #97 as with minor fixes it does work.

**Pull request merged**